test/athens/parser/structure_test.clj

```diff
@@ -0,0 +1,171 @@
+(ns athens.parser.structure-test
+  (:require
+    [athens.parser.structure :as sut]
+    [clojure.test :as t :refer [deftest is testing]]))
+
+
+(defmacro parses-to
```

Comment:
Nit: would be nice for have this macro in another ns, like `athens.common.utils` (which is already cljc and has a macro), and then this ns could be cljc.

Reply:
moved it to `athens.common.utils`